docs/sphinx/source/contributing/style_guide.rst

```diff
@@ -70,6 +70,27 @@ the ``docs/readthedocs.org:pvlib-python`` link within the checks
 status box at the bottom of the pull request.
 
 
+.. _documentation-units:
+
+Parameter names and units
+-------------------------
+
+When specifying parameters and their units, please follow these guidelines:
+
+- Use the recommended parameter name and units by :ref:`nomenclature` where possible.
+- Enclose units in square brackets, e.g. ``[W]``.
+- Use unicode superscripts symbols for exponents, e.g. ``m²``.
```

Comment:
```suggestion
- Use unicode superscripts symbols for exponents, e.g. ``m⁻²``.
```
I think this example would benefit from having a negative exponent to demonstrate that we don't use slashes for units in the denominator.